src/account/get-dispenser-account.ts

```diff
@@ -0,0 +1,22 @@
+import algosdk from 'algosdk'
+import { getLocalNetDispenserAccount } from '../localnet/get-localnet-dispenser-account'
+import { isLocalNet } from '../localnet/is-localnet'
+import { DISPENSER_ACCOUNT } from '../types/account'
+import { getAccount } from './get-account'
+
+import Algodv2 = algosdk.Algodv2
```

Comment:
Now we're using rollup, do we still need this weird format for importing from algosdk?

Reply:
Good question. Let me check.

Reply:
I have confirmed that we can't remove this. The reason we had to make this change was due to algosdk exporting a malformed ESM output, so we need to wait until v3 is released, where this issues is fixed.

See https://github.com/algorandfoundation/algokit-utils-ts/blob/main/.eslintrc#L11 for an explanation.